<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackingParameters</class>
 <widget class="QWidget" name="StackingParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stacking</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Stacking Mode</string>
     </property>
     <widget class="QLabel" name="xackgroundCalibration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="debloom">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>671</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Debloom</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="hotPixels">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Hot pixel detection and removal</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="badColumns">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>306</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bad column detection and removal</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="darkOptimisation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dark Optimisation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deBloomSettings">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>260</y>
        <width>70</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="useDarkFactor">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>143</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dark Multiplication Factor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="darkMultiplicationFactor">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="modeAverage">
         <property name="text">
          <string>Average</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QRadioButton" name="modeEWA">
         <property name="text">
          <string>Entropy Weighted Average
(High Dynamic Range)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="modeMedian">
         <property name="text">
          <string>Median</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QRadioButton" name="modeMaximum">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="modeKS">
         <property name="text">
          <string>Kappa-Sigma clipping</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="modeMKS">
         <property name="text">
          <string>Median Kappa-Sigma clipping</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="modeAAWA">
         <property name="text">
          <string>Auto Adaptive Weighted Average</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="staticKappa">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Kappa:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="kappa">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="staticIterations">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Number of iterations:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="iterations">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="backgroundCalibration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color:blue;
text-align:left</string>
      </property>
      <property name="text">
       <string>No Background Calibration</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>modeAverage</tabstop>
  <tabstop>modeMedian</tabstop>
  <tabstop>modeKS</tabstop>
  <tabstop>modeMKS</tabstop>
  <tabstop>modeAAWA</tabstop>
  <tabstop>modeEWA</tabstop>
  <tabstop>modeMaximum</tabstop>
  <tabstop>hotPixels</tabstop>
  <tabstop>badColumns</tabstop>
  <tabstop>darkOptimisation</tabstop>
  <tabstop>debloom</tabstop>
  <tabstop>useDarkFactor</tabstop>
  <tabstop>darkMultiplicationFactor</tabstop>
  <tabstop>deBloomSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
